### ISSUE TITLE
No Response

### ISSUE THREAD

**Issue by MacFlagg** [samrum/homebridge-onstar#25]
The plugin worked until recently. Now, if a tile is activated in the Home app, it says no response.

Any ideas on how to get it working again?

**Comment by samrum** [comment]
Is it only this plugin that's not working?
Probably just just try restarting homebridge and/or the device you're using as your home hub. Could also try toggling the home hub feature off and on on the device.

**Comment by MacFlagg** [comment]
I have already tried all the basic troubleshooting steps and they did not work. The only thing I’ve not tried is deleting the plugin and reinstalling. However, I was hoping to learn how to troubleshoot the plugin before taking such a drastic measure.

I have zero experience coding, so I get these plugins working by closely following the provided examples. I absolutely don’t understand the code or why in doing what I’m doing. I just know it works, most of the time. Therefore, when something breaks, I don’t have the tools to fix the problem, unless I can find an example from someone with similar issues or I get help. 

I am trying to learn how the plugins I use work and how to troubleshoot and fix them myself. But each plugin seems to be it’s own adventure.

One strange thing that has happened with the plugin, is twice now in the middle of the night while I was sleeping, I get a notification from the Home App that my car doors had been unlocked. This would occur many, many hours after a failed attempt of using the Home App or Siri to operate my vehicle.

**Comment by MacFlagg** [comment]
![ECBB8994-45F2-455E-99BD-805A19E32F61](https://user-images.githubusercontent.com/10389407/76145958-94ec6700-605c-11ea-860a-6ecbaf178b5a.png)

Whenever a tile is pressed, it says “no response” across the bottom of the tile and gets the (!) symbol in the top right hand corner.

**Comment by ukypayne** [comment]
What does the home bridge log say when you press the button? It will usually spit out an actual error.

**Comment by MacFlagg** [comment]
![E1DE38A1-72F9-4591-AA92-A67073453E30](https://user-images.githubusercontent.com/10389407/76155027-5a161d80-60b4-11ea-971d-528def10759d.jpeg)

Here is the error log.

I don’t code, so I have no idea what anything on the log means.

**Comment by samrum** [comment]
Does it always show No Response? Even when you are on the same network as your home hub?

**Comment by MacFlagg** [comment]
Yes

Dayton

**Comment by samrum** [comment]
Do the buttons always show No Response? Or does it end up in that state that after you attempt to toggle one of the buttons and eventually let you toggle them again?

**Comment by MacFlagg** [comment]
It will say no response and then, after some time it will disappear. 

I’ve deleted the plugin and reinstalled it. 

Now, instead of internal server error 500, I get a “you don’t have permission”.

I think I give up.

Dayton

**Comment by samrum** [comment]
😞 Can you share the bit of log (redacted) with that? Don't think I've seen that one before.

**Comment by MacFlagg** [comment]
![IMG_0216](https://user-images.githubusercontent.com/10389407/76760727-8ab91f80-6764-11ea-85f0-6ce02e30e3a0.jpeg)

**Comment by MacFlagg** [comment]
I can log into the MyChevrolet app and I can use the key fob feature without issue.

**Comment by MacFlagg** [comment]
![IMG_0217](https://user-images.githubusercontent.com/10389407/76762726-34e67680-6768-11ea-974f-96d93de7cfe5.jpeg)

I finally got it working. After deleting and reinstalling the plugin 3 times, it finally started working. Since I’ve been copying and pasting all my credentials from 1Password, I know that all that information was added to the plugin correctly each time.

I’ve been getting my UUID from - https://www.uuidgenerator.net/version4

I’m running on HOOBS 3.1.22

**Comment by MacFlagg** [comment]
![IMG_0217](https://user-images.githubusercontent.com/10389407/76762726-34e67680-6768-11ea-974f-96d93de7cfe5.jpeg)

I finally got it working. After deleting and reinstalling the plugin 3 times, it finally started working. Since I’ve been copying and pasting all my credentials from 1Password, I know that all that information was added to the plugin correctly.

I’ve been getting my UUID from - https://www.uuidgenerator.net/version4
